clients/cli/build/update_brew.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eo pipefail
+
+brew_dir=$(mktemp -d -t ci-XXXXXXXXXX)
+git clone --depth 1 https://$API_TOKEN_GITHUB@github.com/phrase/homebrew-brewed.git $brew_dir &> /dev/null
+
+DIST_DIR="./dist"
+FILES_FOR_BREW=("phrase_linux_386.tar.gz" "phrase_linux_amd64.tar.gz" "phrase_macosx_amd64.tar.gz")
```

Comment:
I would have probably just used 
`for file in phrase_linux_386.tar.gz phrase_linux_amd64.tar.gz phrase_macosx_amd64.tar.gz; do`
or maybe even `ls ${DIST_DIR}/phrase_*.tar.gz`? Only a matter of time when we also need `phrase_macosx_arm.tar.gz`.

Reply:
Not all files are needed, for example windows is skipped.